vehicle_model:
  simple_bicycle_model:
  # some made up values
    kinematics:
      lr: 0.791
      lf: 0.809
      sf: 1.2
      sr: 1.2
    tire:
      Blat: 9.63
      Clat: -1.39
      Dlat: 1.6
      Elat: 1.0
    aero:
      cla: 3.7
      cda: 1.1
      aeroArea: 1.1
    m: 275
    Izz: 111.0
    wheelRadius: 0.258
    gearRatio: 4.0
    innerSteeringRatio: 0.255625
    outerSteeringRaito: 0.20375
    nominalVoltageTS: 550.0
    powerGroundForce: 700.0
    powertrainEfficiency: 0.95